### PR TITLE
AAP-19133 Update existing modules for organizations for 2.5 RBAC changes

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR includes RBAC changes for 2.5 for content on Organizations.

### CODE REVIEW

File: downstream/modules/platform/proc-controller-add-organization-user.adoc [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
 +
-image:organizations-add-users-for-example-organization.png[Add roles]
+include::snippets/snip-gw-roles-note-multiple-components.adoc
 +
-In this example, two users have been selected.
-. Select the role you want the selected user or team to have. 
-Scroll down for a complete list of roles. 
-Different resources have different options available.
+. Click btn:[Next] to review the roles settings.
+. Click btn:[Finish] to apply the roles to the selected users, and to add them as members. The Add users dialog displays the updated roles assigned for each user.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-add-organization-user.adoc:18]: The * Add Users * dialog

File: downstream/modules/platform/proc-controller-add-organization-user.adoc [hunk 2]
```diff
@@ -1,37 +1,26 @@
...
-= Add a User or Team
-
-To add a user or team to an organization, the user or team must already exist. 
-
-For more information, see xref:proc-controller-creating-a-user[Creating a User] and xref:proc-controller-creating-a-team[Creating a Team].
-
-To add existing users or team to the Organization:
+Adding a user to an organization
 
+You can provide a user with access to an organization by adding them to the organization and managing the roles associated with the user. To add a user to an organization, the user must already exist. For more information, see Create a user.
```
Comment (ianf77) [inline downstream/modules/platform/proc-controller-add-organization-user.adoc:5]: Does this line need an xref?

**Review by ianf77 — COMMENTED** [review]
Couple of comments

File: downstream/modules/platform/proc-controller-add-organization-user.adoc (hunk 2 above)
Comment (ianf77) [inline downstream/modules/platform/proc-controller-add-organization-user.adoc:5]: Should this be an xref:

Comment (dcdacosta) [reply downstream/modules/platform/proc-controller-add-organization-user.adoc:5]: Good catch, thank you!

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**